pkg/slack/slack.go

```diff
@@ -169,7 +169,8 @@ func (b *Bot) SupportedCommands() []parser.BotCommand {
 func GetUserName(client *slack.Client, userID string) string {
 	user, err := client.GetUserInfo(userID)
 	if err != nil {
```

Comment:
Curious, what does the error say is wrong?  We've obviously never seen this case before...

Reply:
This is using my own slack app (with a RequestURL that ends in `/slack/events-endpoint` ) and I see this:

```
W0717 16:03:10.518868   17610 slack.go:172] Failed to get the User Info for UserID: UF32A653J
```

Followed by:

```
"runtime error: invalid memory address or nil pointer dereference"
Stack:
	 4  0x000000010c4e06e4 in github.com/openshift/ci-chat-bot/pkg/slack.GetUserName
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/pkg/slack/slack.go:174
	 5  0x000000010c4d6bda in github.com/openshift/ci-chat-bot/pkg/slack.LaunchCluster
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/pkg/slack/actions.go:18
	 6  0x000000010c4d5090 in github.com/openshift/ci-chat-bot/pkg/slack/parser.(*botCommand).Execute
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/pkg/slack/parser/commandParser.go:38
	 7  0x000000010c4f5799 in github.com/openshift/ci-chat-bot/pkg/slack/events/messages.Handle.func1
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/pkg/slack/events/messages/message_handler.go:53
	 8  0x000000010c4ecaeb in github.com/openshift/ci-chat-bot/pkg/slack/events.(*partialHandler).Handle
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/pkg/slack/events/handler.go:48
	 9  0x000000010c4ecee9 in github.com/openshift/ci-chat-bot/pkg/slack/events.MultiHandler.func1
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/pkg/slack/events/handler.go:86
	10  0x000000010c4ec91c in github.com/openshift/ci-chat-bot/pkg/slack/events.(*handler).Handle
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/pkg/slack/events/handler.go:20
	11  0x000000010c522f1b in main.handleEvent.func1.1
	     at /Volumes/EnvoyProSX/mygit/dperique/NG/Dennis/Redhat/ci-chat-bot/cmd/ci-chat-bot/slack.go:110
	(truncated)
```

Reply:
What Slack Workspace are you connecting to?  Your own?

Reply:
my own

Reply:
If you print the err that's returned, what does it say?

Reply:
I pushed a change to print the `err` and get:

```
W0717 16:26:50.131979   23732 slack.go:172] Failed to get the User Info for UserID: UF32A653J; missing_scope
```